Core/GraphQL/GraphQLFiles/sell_getProductsById.graphql

```diff
@@ -0,0 +1,5 @@
+query GetProduct($id: ID!) {
```

Comment:
The file name should be `sell_getProductById.graphql`. And could you change the query name to `getProductById($id: ID!)` please